src/scss/includes/components/poiItem.scss

```diff
@@ -0,0 +1,34 @@
+.PoiItem {
+  padding: 16px 30px;
+  cursor: pointer;
+  display: flex;
+
+  @media (max-width: 640px) {
+    padding: 12px 16px;
+    display: flex;
+  }
+
+  .poiTitleImage {
+    width: 65px;
+    height: 65px;
+  }
+
+  &-address {
+    text-overflow: ellipsis;
+    max-width: 210px;
+    white-space: nowrap;
+    overflow: hidden;
+    margin-bottom: 4px;
+  }
+
+  &-reviews {
+    display: inline-flex;
+    margin-bottom: 4px;
+  }
+
+  &-right {
```

Comment:
Not sure I understand the rules on this element.
Do we want to center vertically the image? What does the `margin-left: auto` do?
BTW, on the screenshots, the icons of POI without an image isn't aligned with the image of the other POIs.

Reply:
The name `right` might be confusing ? It's the right section of the item (the card). Hence, `margin-left: auto` will "push" the element to the right.

Reply:
Thanks for the explanation. That's the kind of case where I prefer using `space-between` for pushing to the right, and the `margin-left` to control the mininum margin with the element on the left.

Reply:
Thanks for the tip, I set space-between and added a margin-left of 16px on the "right" class. Is the name ".right" ok for you ? May I rename it rightSection or something else ?

Reply:
Thanks for the change. And `.poiItem-right` is ok.

Reply:
The alignement problem of icons vs. images 
![Capture d’écran de 2020-06-04 12-03-44](https://user-images.githubusercontent.com/243653/83744442-6df20100-a65c-11ea-8b31-75ce6b829447.png)